src/platform/imx8/include/platform/lib/memory.h

```diff
@@ -14,7 +14,7 @@
 
 
 /* data cache line alignment */
-#define PLATFORM_DCACHE_ALIGN	DCACHE_LINE_SIZE
+#define PLATFORM_DCACHE_ALIGN	sizeof(void *)
```

Comment:
Does this build properly? This header might possibly be included by stuff that isn't C code I think...

Reply:
@paulstelian97 thanks Paul very good point! This builds properly! You can see that only HEAP_BUF_ALIGNMENT is used from non C code (e.g linker files). This is also the way intel non-cavs platforms are handling the alignment.